_includes/install_command.html

```diff
@@ -0,0 +1,10 @@
+{%- capture code -%} 
+brew {{include.modifier}}install {{include.item}} 
+{%- endcapture -%} 
+
+Install command:
+<div class="copyable">
+    {% highlight bash %} $ {{ code }} {% endhighlight %}
+</div>
+
+<script src="//brew.sh/assets/js/copy.js"></script>
```

Comment:
Needs a newline at the bottom here. Also, I think having this script tag in the base layout on brew.sh would mean it's not needed here.

Reply:
`<script src="{{ "/assets/js/copy.js" | relative_url }}" async></script>`

This should be added to the base.html in brew.sh?